realm/src/main/java/io/realm/internal/TableQuery.java

```diff
@@ -480,7 +480,16 @@ public long find() {
         return nativeFind(nativePtr, 0);
     }
 
-    protected native long nativeFind(long nativeQueryPtr, long fromTableRow);
+    public long findWithHandover(long bgSharedGroupPtr, long nativeReplicationPtr, long ptrQuery) {
```

Comment:
I'll add it.
other method are also missing Javadoc btw (find, contains, etc.) it's an internal package after all